docs/reference/NuGet-Client-SDK.md

```diff
@@ -9,14 +9,46 @@ ms.topic: conceptual
 
 # NuGet Client SDK
 
-The *NuGet Client SDK* refers to a group of NuGet packages centered around [NuGet.Commands](https://www.nuget.org/packages/NuGet.Commands), [NuGet.Packaging](https://www.nuget.org/packages/NuGet.Packaging), and [NuGet.Protocol](https://www.nuget.org/packages/NuGet.Protocol). These packages replace the earlier [NuGet.Core](https://www.nuget.org/packages/NuGet.Core/) library.
+The *NuGet Client SDK* refers to a group of NuGet packages centered around the [NuGet.Protocol](https://www.nuget.org/packages/NuGet.Protocol) and [NuGet.Packaging](https://www.nuget.org/packages/NuGet.Packaging) packages from the [NuGet/NuGet.Client](https://github.com/NuGet/NuGet.Client) GitHub repository.
```

Comment:
We removed the reference to NuGet.Commands but do not mention NuGet.Packaging below. From the docs, it seems like the recommended package to use is NuGet.Protocol. NuGet.Packaging is a dependency of NuGet.Protocol but I think it's unclear to the reader what this relationship is or if the NuGet.Packaging package is even usable on its own.

Maybe a short blurb about NuGet.Packaging is in order? Something like NuGet.Packaging is useful for interacting with NuGet packages already on your computer and NuGet.Protocol allows interacting with HTTP and file-based feeds. 

Alternatively we can remove the reference to NuGet.Packaging as well. NuGet.Frameworks and NuGet.Versioning are also in the graph but unmentioned.

Reply:
Added